Display the hostname when accepting a message via SMTP (other MTAs log
this acceptance line, so we can tell which SMTP server a message hit.

This patch is no longer maintained.

This patch is distributed under the same terms as qmail-ldap.

Author: John Morrissey <jwm@horde.net>

--- qmail-1.03.stock/qmail-smtpd.c.orig	Fri Jan  4 11:25:54 2002
+++ qmail-1.03.stock/qmail-smtpd.c	Fri Jan  4 11:31:37 2002
@@ -974,7 +974,10 @@
 char accept_buf[FMT_ULONG];
 void acceptmessage(qp) unsigned long qp;
 {
+  char hostname[64] = {'\0'};
   datetime_sec when;
+
+  gethostname(hostname, sizeof(hostname) - 1);
   when = now();
   out("250 ok ");
   accept_buf[fmt_ulong(accept_buf,(unsigned long) when)] = 0;
@@ -983,6 +986,8 @@
   out(" qp ");
   accept_buf[fmt_ulong(accept_buf,qp)] = 0;
   out(accept_buf);
+  out(" at ");
+  out(hostname);
   out("\r\n");
   logstring(2,"qp"); logstring(2,accept_buf); logflush(2);
 }
